lms/djangoapps/instructor/features/data_download.py

```diff
@@ -18,7 +18,7 @@ def find_student_profile_table(step):  # pylint: disable=unused-argument
     world.wait_for_visible('#data-student-profiles-table')
 
     # Wait for the data table to be populated
-    world.wait_for(lambda _: world.css_text('#data-student-profiles-table') not in [u'', u'Loading...'])
+    world.wait_for(lambda _: world.css_text('#data-student-profiles-table') not in [u'', u'Loading...', u'Loading'])
```

Comment:
I don't see any hits when searching for "Loading..." in the LMS. I think you can just check for "Loading". Looks like all ellipses were removed by Bertrand Marron on 1/5/2015.